_data/community_neurons.yml

```diff
@@ -5,7 +5,13 @@
   demos:
     - demo_url: https://www.youtube.com/embed/q_giVwsF-6A
       synapse: fr_synapse.yml
-
+      
+- neuron_name: Answer of everything
+  description: Get answers of all kind of questions
```

Comment:
I would suggest to add a clue on the fact that use DuckDuck or wolframe, etc... don't you think?

Reply:
Yes you are right, it's a bit more meaningful then.